README.md

```diff
@@ -585,6 +585,7 @@ To configure a Memcache instance use the following environment variables instead
 1. `MEMCACHE_HOST_PORT=`: a comma separated list of hostname:port pairs for memcache nodes.
 1. `BACKEND_TYPE=memcache`
 1. `CACHE_KEY_PREFIX`: a string to prepend to all cache keys
+1. `MEMCACHE_MAX_IDLE_CONNS=-1`: the maximum number of idle TCP connections per memcache node, if `-1` this will use the default of the underlying library (currently 2)
```

Comment:
This could be confusing for users that setting value to `-1` translates to `2`. Can we instead allows users to configure values within [1, N] range and internally check if this variable is not set, then don't propagate it to memcache client

Reply:
Could you explain why users may need to tune this parameter?